eng/pipelines/bicep-warnings.yml

```diff
@@ -0,0 +1,47 @@
+trigger: none
+
+pr:
+  paths:
+    include:
+      - eng/pipelines/bicep-warnings.yml
+
+stages:
+  - stage: LintBicepFiles
+    displayName: Lint Bicep Files
+
+    pool:
+      vmImage: ubuntu-22.04
+
+    jobs:
+      - job: LintBicep
+        timeoutInMinutes: 20
+        steps:
+          - task: AzureCLI@2
+            displayName: 'Install Bicep CLI and Lint Bicep Files'
+            inputs:
+              azureSubscription: 'opensource-api-connection'
```

Comment:
Do you need this level of permission to install tool? If you don't need to be logged in then I would switch this to just be a script task.

Reply:
I suppose not, changed.